outputs.tf

```diff
@@ -2,6 +2,11 @@ output "function_arn" {
   value = aws_lambda_function.ses_redirecter.arn
 }
 
+output "record_mx_record" {
+  value = join(" ", [aws_route53_record.mx.name, aws_route53_record.mx.records[0]])
```

Comment:
this appears to still reference route53. I think the records needed would come from an ses resource right?

Reply:
Oops. I didn't re-look at that one.